components/control/Link/Link.stories.ts

```diff
@@ -0,0 +1,47 @@
+import { StoryObj } from '@storybook/web-components';
+import { html } from 'lit';
+
+import './Link';
+
+export default {
+  component: 'diamond-link',
+  parameters: {
+    docs: {
+      description: {
+        component: 'Provides link styles for anchor or button elements.',
```

Comment:
Why?

Reply:
Sometimes you want something that looks like a link, but is semantically a button. Same as when you want something that looks like a button, but is semantically a link.

Reply:
And then sometimes you want non underlined links too.